src/button-react/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createButton from '../button';
+
+export default class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.button = React.createRef();
+    this.componentDidMount = this.componentDidUpdate = () =>
+      createButton(this.button.current, this.props);
+  }
+
+  render() {
+    return React.createElement('div', { ref: this.button });
+  }
+};
+
+Button.propTypes = {
+  size: PropTypes.oneOf(['icon', 'small', 'medium', 'large']),
+  url: PropTypes.string,
+  account: PropTypes.string,
+};
```

Comment:
Please, setup your IDE to handle [.editorconfig](https://github.com/aeternity/superhero-utils/blob/b58a16c41267c97d5d7ee9c0beee4f692290c891/.editorconfig#L9) files.

Reply:
done, thanks